app/src/processing/app/EditorConsole.java

```diff
@@ -110,18 +113,86 @@ public EditorConsole() {
     setMinimumSize(new Dimension(100, (height * lines)));
 
     EditorConsole.init(stdOutStyle, System.out, stdErrStyle, System.err);
+
+    // Add "CTRL scroll" hotkey for font size adjustment.
+    consoleTextPane.addMouseWheelListener((MouseWheelEvent e) -> {
+      if (e.isControlDown()) {
+        if (e.getWheelRotation() < 0) {
+          base.handleFontSizeChange(1);
+        } else {
+          base.handleFontSizeChange(-1);
+        }
+      } else {
+        e.getComponent().getParent().dispatchEvent(e);
+      }
+    });
+
+    // Add "CTRL (SHIFT) =/+" and "CTRL -" hotkeys for font size adjustment.
+    consoleTextPane.addKeyListener(new KeyAdapter() {
+      @Override
+      public void keyPressed(KeyEvent e) {
+        if (e.getModifiersEx() == KeyEvent.CTRL_DOWN_MASK
+            || e.getModifiersEx() == (KeyEvent.CTRL_DOWN_MASK | KeyEvent.SHIFT_DOWN_MASK)) {
+          switch (e.getKeyCode()) {
+            case KeyEvent.VK_PLUS:
+            case KeyEvent.VK_EQUALS:
+              base.handleFontSizeChange(1);
+              break;
+            case KeyEvent.VK_MINUS:
+              if (!e.isShiftDown()) {
+                base.handleFontSizeChange(-1);
+              }
+              break;
+          }
+        }
+      }
+    });
```

Comment:
Same for this KeyAdapter, maybe it can be written just once inside Base?

Reply:
Done in 2d593c4.